docs/chaos-engineering/chaos-faults/aws/ec2-network-latency.md

```diff
@@ -0,0 +1,309 @@
+kind: Secret
+metadata:
+  name: cloud-secret
+type: Opaque
+stringData:
+  cloud_config.yml: |-
+    # Add the cloud AWS credentials respectively
+    [default]
+    aws_access_key_id = XXXXXXXXXXXXXXXXXXX
+    aws_secret_access_key = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+```
+
+- If you change the secret name then please also update the `experiment.yml` ENV values for deriving the respective data from the secret. Also account for the path at which this secret is mounted as a file in the manifest ENV `AWS_SHARED_CREDENTIALS_FILE`.
+
+### NOTE
+You can pass the VM credentials as secrets or as an ChaosEngine ENV variable.
+:::
+
+    
+## Default Validations
+:::info
+- EC2 instance should be in healthy state.
+:::
+
+## Experiment Tunables
+<details>
+    <summary>Check the Experiment Tunables</summary>
+    <h2>Mandatory Fields</h2>
+    <table>
+      <tr>
+        <th> Variables </th>
+        <th> Description </th>
+        <th> Notes </th>
+      </tr>
+      <tr>
+        <td> EC2_INSTANCE_ID </td>
+        <td> ID of the target EC2 instance </td>
+        <td> For example: <code>i-044d3cb4b03b8af1f</code> </td>
+      </tr>
+      <tr>
+        <td> REGION </td>
+        <td> The AWS region ID where the EC2 instance has been created </td>
+        <td> For example: <code>us-east-1</code> </td>
+      </tr>
+    </table>
+    <h2>Optional Fields</h2>
+    <table>
+        <tr>
+            <th> Variables </th>
+            <th> Description </th>
+            <th> Notes </th>
+        </tr>
+        <tr>
+            <td> TOTAL_CHAOS_DURATION </td>
+            <td> The total time duration for chaos insertion (sec) </td>
+            <td> Defaults to 30s </td>
+        </tr>
+        <tr>
+            <td> CHAOS_INTERVAL </td>
+            <td> The interval (in sec) between successive instance termination </td>
+            <td> Defaults to 30s </td>
+        </tr>
+        <tr>
+            <td> AWS_SHARED_CREDENTIALS_FILE </td>
+            <td> Provide the path for aws secret credentials</td>
+            <td> Defaults to <code>/tmp/cloud_config.yml</code> </td>
+        </tr>
+        <tr>
+            <td> DOCUMENT_NAME </td>
+            <td> Provide the name of addded ssm docs (if not using the in-build docs)</td>
+            <td> Default to Litmus-AWS-SSM-Docs-For-EC2-Network-Latency </td>
+        </tr>
+        <tr>
+            <td> DOCUMENT_FORMAT </td>
+            <td> Provide the format of the ssm docs. It can be YAML or JSON</td>
+            <td> Defaults to YAML</td>
+        </tr>
+        <tr>
+            <td> DOCUMENT_TYPE </td>
+            <td> Provide the document type of added ssm docs (if not using the default docs)</td>
+            <td> Defaults to Command </td>
+        </tr>
+        <tr>
+            <td> DOCUMENT_PATH </td>
+            <td> Provide the document path if added using configmaps</td>
+            <td> Defaults to Litmus-AWS-SSM-Docs-For-EC2-Network-Latency.yml </td>
+        </tr>
```

Comment:
```suggestion
```
We can remove these as the default document will be used for these experiments, if the user wants to use custom document they can use the SSM Command experiment ( as discussed recently)
cc: @uditgaurav @Jonsy13